atlasexec/atlas_schema_test.go

```diff
@@ -249,6 +249,15 @@ func TestSchema_Plan(t *testing.T) {
 			},
 			args: `schema plan --format {{ json . }} --from 1,2 --to 2,3 --auto-approve`,
 		},
+		{
+			name: "with from to and directives",
+			params: &atlasexec.SchemaPlanParams{
+				From:       []string{"1", "2"},
+				To:         []string{"2", "3"},
+				Directives: []string{"atlas:nolint", "\"atlas:txmode none\""},
```

Comment:
I think it should be `Directives: []string{"atlas:nolint", "atlas:txmode none"},`

Quote string should handle by the SDK.